features/search/FilterDialog.tsx

```diff
@@ -157,6 +157,12 @@ export default function FilterDialog({
                         searchFilters.active.lng ?? 0,
                         searchFilters.active.lat ?? 0
                       ),
+                      bbox: searchFilters.active.bbox as [
+                        number,
+                        number,
+                        number,
+                        number
+                      ],
```

Comment:
I think it would be safer to default to `[0, 0, 0, 0]` as opposed to type asserting here since `bbox` could be undefined if the value doesn't exist in the filters (from the query params)?

Reply:
hmm, value came from the API and then is set in the query param, so in principle is always existing

Reply:
That doesn't seem like a safe assumption given we allow people to open the dialog and apply the filters (closes the dialog) without enforcing location as a required field 🤔

Reply:
updated :)